From: Sebastian Spaeth <Sebastian@SSpaeth.de>
Date: Thu, 27 Oct 2022 19:52:11 +0200
Subject: [PATCH] Do not install the sv2png script

It is not part of the .png files, it is not used during runtime  and lintian
correctly complains about a binary in such a location.
---
 qml/CMakeLists.txt | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/qml/CMakeLists.txt b/qml/CMakeLists.txt
index 2e47657..239f637 100644
--- a/qml/CMakeLists.txt
+++ b/qml/CMakeLists.txt
@@ -1,6 +1,7 @@
 install(FILES ${QML_SRC} DESTINATION ${DATADIR}/qml)
 install(FILES ${QML_JS} DESTINATION ${DATADIR}/qml/js)
-install(DIRECTORY icons DESTINATION ${DATADIR}/qml)
+install(DIRECTORY icons DESTINATION ${DATADIR}/qml
+        PATTERN svg2png EXCLUDE)
 
 # We can't just install entire directories, as the kirigami flavor includes symlinked files
 # To remedy this we dereference them first
